### PR TITLE
Mailchimp styles

### PR DESCRIPTION
**Pull request by fjp2cornell**
Testing on featuretest

### CODE REVIEW

**Comment by fjp2cornell** [comment]
This branch is now ready for merging.

File: docroot/sites/all/themes/ilr_theme/scss/layout/_base.scss [hunk 1]
```diff
@@ -62,19 +60,19 @@
...
-      @extend %one-column;
-    }
-  }
+  //@todo: duplicate styles - delete
+  // #content-bottom {
+  //   clear: both;
+  //   article {
+  //     @extend %one-column;
+  //   }
+  // }
```
Comment (a-fro) [inline docroot/sites/all/themes/ilr_theme/scss/layout/_base.scss]: Why leave code that's commented out?

**Comment by a-fro** [comment]
Thanks John. Please rename the styles to "_mailchimp.scss" (without the underscore), since it's one word and that's the convention.

File: docroot/sites/all/themes/ilr_theme/scss/layout/_base.scss (hunk 1 above)
Comment (fjp2cornell) [inline docroot/sites/all/themes/ilr_theme/scss/layout/_base.scss]: Must have been in there for a while but I can remove it

**Comment by fjp2cornell** [comment]
Changes have been made and pushed.

**Comment by a-fro** [comment]
Nice. Looks great! Thanks John.

**Pull request merged**